### PR TITLE
Minor updates to 2.1.45

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/preferences.md [hunk 1]
```diff
@@ -50,18 +50,19 @@ deck.
...
-The Anki 2.1 scheduler is documented here:
-<https://faqs.ankiweb.net/the-anki-2.1-scheduler.html>
-
-**Mix new cards and reviews**:
-This drop-down box controls when new cards are shown: either mixed
-with, before, or after all reviews.
+**Show next review time above answer buttons**  
+Useful to know how far in the future your cards are being pushed.
+
+**Show remaining card count review**
```
Comment (dae) [inline src/preferences.md:56]:
```suggestion
**Show remaining card count during review**  
```

File: src/preferences.md [hunk 2]
```diff
@@ -50,18 +50,19 @@ deck.
...
-This drop-down box controls when new cards are shown: either mixed
-with, before, or after all reviews.
+**Show next review time above answer buttons**  
+Useful to know how far in the future your cards are being pushed.
+
+**Show remaining card count review**  
+Disable this option to hide the card count at the bottom of the screen.
+
+**Legacy timezone handling**  
+Please, see this:
```
Comment (dae) [inline src/preferences.md:60]:
```suggestion
Please see here:
```

"Please, " places a little too much emphasis on the word please, and sounds like you're begging. :-)

File: src/preferences.md [hunk 3]
```diff
@@ -50,18 +50,19 @@ deck.
...
-day delay are shown after normal reviews. If checked, Anki will show them before
-normal reviews instead.
-
-**Anki 2.1 scheduler**  
-The Anki 2.1 scheduler is documented here:
-<https://faqs.ankiweb.net/the-anki-2.1-scheduler.html>
-
-**Mix new cards and reviews**:
-This drop-down box controls when new cards are shown: either mixed
-with, before, or after all reviews.
```
Comment (dae) [inline src/preferences.md:64]: This setting will still be shown when the v1/v2 schedulers are enabled, so it's too early to remove this. Same for "show learning cards with larger steps" (v2)

**Review by dae — COMMENTED** [review]
Thanks cqg! Some comments below.

File: src/preferences.md (hunk 3 above)
Comment (cqg2) [reply src/preferences.md:64]: Done, thanks!

**Comment by dae** [comment]
Thanks cqg!

**Pull request merged**